crates/graphql-composition/tests/composition/entity_interfaces_with_requires/federated.graphql

```diff
@@ -32,7 +32,7 @@ type Book implements Media
     @join__implements(graph: A, interface: "Media")
 {
     id: ID!
-    reviews: [Review!]! @join__field(graph: B, requires: "title")
+    reviews: [Review!]! @join__field
```

Comment:
Is it expected here that the requires is gone?

Reply:
Yes, because it's not `Book.reviews` that has a requirement, but `Media.reviews`.